src/timer.c

```diff
@@ -130,6 +130,13 @@ uint64_t uv_timer_get_repeat(const uv_timer_t* handle) {
 }
 
 
+uint64_t uv_timer_get_due_in(const uv_timer_t* handle) {
+  uint64_t due = handle->timeout;
+  uint64_t now = uv_now(handle->loop);
+  return due - now;
```

Comment:
Style:
```suggestion
  return handle->timeout - handle->loop->time;
```
I mean, assigning to temporaries is okay but going through `uv_now()` isn't necessary.

Substance:

It underflows for expired timers because `handle->timeout < loop->time`. I'd return `0` in that case, although that won't let you distinguish between an expired timer and one that is expiring right now.